.eslintignore

```diff
@@ -4,4 +4,5 @@ speech/
 training_data/
 ui/ibm/watson-speech.min.js
 ui_config/
-/*.js
```

Comment:
@doconnor78 I think this is looking at "/*.js" , not "*.js", so it's really only being ignored if it's in the root directory (app.js, gulpfile.js, server.js).